kimera_pgmo/CMakeLists.txt

```diff
@@ -50,12 +50,20 @@ target_include_directories(
          ${PCL_INCLUDE_DIRS})
 target_link_libraries(
   ${PROJECT_NAME}
-  PUBLIC ${PCL_LIBRARIES} Eigen3::Eigen kimera_rpgo::kimera_rpgo gtsam
-         pose_graph_tools::pose_graph_tools spatial_hash::spatial_hash
-  PRIVATE "$<BUILD_INTERFACE:config_utilities::config_utilities>")
+  PUBLIC ${PCL_LIBRARIES}
+         Eigen3::Eigen
+         kimera_rpgo::kimera_rpgo
+         gtsam
+         pose_graph_tools::pose_graph_tools
+         spatial_hash::spatial_hash
+         config_utilities::config_utilities
```

Comment:
This should be paired with an update to `find_dependencies(config_utilities REQUIRED)` in the pgmo cmake config that gets installed (and removing the `find_package(config_utilities REQUIRED)` in `kimera_pgmo_ros`). I think when I added it originally I managed to avoid any includes of `config_utilities` in headers so it was okay for it to be private, but we've since added it elsewhere